tests/cunit/test_iosystem2_simple.c

```diff
@@ -26,8 +26,8 @@
 #define BASE 0
 #define REARRANGER 1
 
-/* Ten megabytes. */
-#define TEN_MEG 10485760
+/* Sixty-four megabytes. */
+#define SIXTY_FOUR_MEG 67108864
```

Comment:
Just a suggestion, can we ignore the old code's naming style and rename this to SIXTY_FOUR_MB ?

Reply:
Sure, I will rename it.